openshift/packageAnalyzerSyncJob-template.yaml

```diff
@@ -0,0 +1,161 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: graph-sync-job-package-analyzer
+  annotations:
+    description: "Thoth: Graph Sync Job for package-analyzer documents"
+    openshift.io/display-name: "Thoth: Graph Sync Job Package-Analyzer"
+    version: 0.6.1
+    tags: thoth,ai-stacks,aistacks,graph-sync-job
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to deploy Thoth's Graph Sync Job for Package-Analyzers on OpenShift.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+  labels:
+    template: graph-sync-job-package-analyzer
+    app: thoth
+    component: graph-sync
+    graph-sync-type: package-analyzer
+
+parameters:
+  - description: Registry the ImageStream to be use lives in
+    displayName: ImageStream Registry
+    required: true
+    name: IMAGE_STREAM_REGISTRY
+    value: "docker-registry.default.svc:5000"
+
+  - description: Namespace the ImageStream to be use lives in
+    displayName: ImageStream Project
+    required: true
+    name: IMAGE_STREAM_NAMESPACE
+    value: "thoth-infra-stage"
```

Comment:
Package-Analyzer is deployed on **thoth-test-core**, but I can deploy to **thoth-infra-stage** if there is a need.

Reply:
Keep it just in thoth-test-core for now.

Reply:
@fridex I changed it to **thoth-test-core**

Reply:
I meant - keep it deployed only in thoth-test-core. The configuration option you changed is adjusted from our deployment scripts.

Reply:
Got it, let me bring **thoth-infra-stage** back.